packages/react-components/react-toolbar/src/components/Toolbar/useToolbar.ts

```diff
@@ -64,9 +64,23 @@ export const useToolbar_unstable = (props: ToolbarProps, ref: React.Ref<HTMLElem
     },
   );
 
+  const handleRadio: ToggableHandler = useEventCallback(
+    (e: React.MouseEvent | React.KeyboardEvent, name?: string, value?: string, checked?: boolean) => {
+      if (name && value) {
+        checkedValues?.[name];
```

Comment:
```suggestion
```

this line does nothing

Reply:
Done https://github.com/microsoft/fluentui/pull/25343/commits/577c64ae04977f288913393f35eb6c3c917c7d04